etl-pipeline/processes/record_embellisher.py

```diff
@@ -24,36 +32,41 @@ def __init__(self, db_manager: DBManager):
 
         self.record_buffer = RecordBuffer(db_manager=self.db_manager)
 
-    def frbrize_record(self, record: Record) -> Record:
+    def embellish_record(self, record: Record) -> Record:
         self._add_works_for_record(record=record)
 
         self.record_buffer.flush()
 
+        # TODO: change this to embellish_status
```

Comment:
@kylevillegas93 how do you recommend making a change requiring a DB migration like this one?

Reply:
let's leave it as is for now, especially since the classify process is still up and running while we implement the new redesign. feel free to create a ticket for it and we will get to it later